GUIDE.md

```diff
@@ -0,0 +1,181 @@
+      topics/streams/writing-a-stream.md    # for authors of streams
+      topics/streams/index.md               # links to both of the above
+
+In other cases, in order to explain an API, reference documentation may need to
+describe a larger topic. In that case, the topic should be described
+separately, as a document in `topics/`, and linked to from the reference in
+`references/`. This is especially true when multiple references need to
+describe the same topic!
+
+We believe good documentation is fractal. Guides, topics, and reference may all
+repeat that organization internally, so long as it does not interfere with the
+communication of the core idea of the document. For example, it's good to add
+links to reference materials at the end of topic docs, but it would be unwise
+to enumerate a module's entire list of methods inside that same doc. Another
+example: it's great to include *brief* example code in the reference
+documentation, but it is not so great to include a full program there.
+
+To summarize:
+
+* Structure is of paramount importance.
+* Decide on an **audience** and an **idea** you want to convey to that
+audience.
+* If the reader is to follow a process to get to a goal, it's a **guide.**
+* If what you're writing conveys a concept to help the reader in decision
+making, it's a **topic.**
+* When the reader is consulting your document to find a specific piece of
+information, it's **reference.**
+* Each document should clearly convey *one* idea to *one* audience. All
+strategies are in service of that goal.
+
+## How We Write
+
+* Documents are written in markdown files. 
+* Those files should be written in **`lowercase-with-dashes.md`.**
+  * Underscores in filenames are allowed only when they are present in the
+    topic the document will describe (e.g., `child_process`.)
+  * Filenames should be **lowercase**.
+  * Older files may use the `.markdown` extension. These may be ported to `.md`
+    at will. **Prefer `.md` for all new documents.**
+* Documents should be word-wrapped at 80 characters.
+* Documents should not contain trailing whitespace.
+* Mechanical issues, like spelling and grammar, should be identified by tools,
+  insofar as is possible. If not caught by a tool, they should be pointed out by
+  human reviewers.
+* American English spelling is preferred. "Capitalize" vs. "Capitalise",
+  "color" vs. "colour", etc.
+* Though controversial, the [Oxford comma][] is preferred for clarity's sake.
+* Generally avoid personal pronouns in reference documentation ("I", "you",
+  "we".)
+  * Pronouns are acceptable in more colloquial documentation, like guides.
+  * Use **gender-neutral pronouns** and **mass nouns**. Non-comprehensive
+    examples:
+    * **OK**: "they", "their", "them", "folks", "people", "developers", "cats"
+    * **NOT OK**: "his", "hers", "him", "her", "guys", "dudes".
+* Place end-of-sentence punctuation inside wrapping elements — periods go
+  inside parentheses and quotes, not after.
```

Comment:
(The rule appears to be concerned with whether the wrapping element contains a complete clause, like this one does.)